modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -47,14 +47,17 @@ def submit_form_2122 # rubocop:disable Metrics/MethodLength
             power_of_attorney.save!
           end
 
-          # This job only occurs when a Veteran submits a POA request, they are not required to submit a document.
-          ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id) unless header_request?
           if enable_vbms_access?
-            ClaimsApi::VBMSUpdater.perform_async(power_of_attorney.id,
-                                                 target_veteran.participant_id)
+            ClaimsApi::VBMSUpdater.perform_async(power_of_attorney.id, target_veteran.participant_id)
           end
+
           data = power_of_attorney.form_data
-          ClaimsApi::PoaFormBuilderJob.perform_async(power_of_attorney.id) if data['signatures'].present?
+
+          if data['signatures'].present?
```

Comment:
one thought I had around validating this: is this enough, or should we go one level deeper and make sure that `data['signatures']` contains `data['signatures']['veteran']` and `data['signatures']['representative']`? I'm leaning towards adding in the stricter validation

Reply:
I don't think there's any harm in adding that stricter validation.  I think the `validate_json_schema` call will blow up if `signatures` is included, but doesn't include both the nested `veteran` and `representative` keys.  But it also isn't going to hurt anything to be extra thorough here as well.